connect-base/1.3/docker-entrypoint.sh

```diff
@@ -164,6 +164,28 @@ case $1 in
         #
         # Configure the log files ...
         #
+        if [[ -n "$CONNECT_LOG4J_LOGGERS" ]]; then
+            sed -i -r -e "s|^(log4j.rootLogger)=.*|\1=${CONNECT_LOG4J_LOGGERS}|g" $KAFKA_HOME/config/log4j.properties
+            if [[ "$CONNECT_LOG4J_LOGGERS" != *"appender"* ]]; then
```

Comment:
Do we really wnt to remove the appender? What if we configure a non-root logger that would use it.

Reply:
@jpechane yes, so far as I can tell, the appender is a problem for us.  Here is my issue.  Our standard logging uses stdout and send logs to splunk.  Those log files appear in `/var/log/docker-compose.log` and are managed automatically with docker-compose setting I mentioned with `logging`:
```
    logging:
        driver: "json-file"
        options:
            max-file: "5"
            max-size: "10m"
```
This manages log rotation and compression.  The problem is that `appender` also creates duplicate log files in the docker directories `/var/lib/docker` with names of `connect-service.log.*`.  We have no way to manage these (compress/log rotate), and they are creating out of disk problems for us as a result.  We don't understand the need for those files at all.

If you have any other suggestion I don't know about, I'm open to it.  But we don't see the need at all for `appender` in our use, unless I'm missing something.

Reply:
But is not it enough to remove appender from root logger but still keep appender config? IMHO unless it is used the config is harmless

Reply:
That's right.  It was just for cleanliness, so it doesn't matter either way to me.  Would you rather I remove that part?

Reply:
Yeah, 'd prefer to keep the appender definition even it is not used as it might be handy for somebody.